src/main/java/chessmaster/commands/ViewLegendCommand.java

```diff
@@ -0,0 +1,35 @@
+//@@author TriciaBK
+package chessmaster.commands;
+
+import chessmaster.game.ChessBoard;
+import chessmaster.ui.TextUI;
+
+public class ViewLegendCommand extends Command {
+
+    public static final String VIEW_LEGEND_COMMAND_STRING = "legend";
+
```

Comment:
Should use the ChessPiece children classes' static variables for the exact representation. e.g.
```java
String.format("\"%s\" represents a white rook.", Rook.ROOK_WHITE);
```